hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/context/NodeInfo.java

```diff
@@ -53,26 +53,26 @@ public NodeInfo(long selfId, Supplier<AddressBook> book) {
     }
 
     /**
-     * For a staked node, validates presence of a self-account in the address book.
+     * For a non-zero weight node, validates presence of a self-account in the address book.
      *
-     * @throws IllegalStateException if the node is staked but has no account
+     * @throws IllegalStateException if the node has a non-zero weight but has no account
      */
-    public void validateSelfAccountIfStaked() {
-        if (!isSelfZeroStake() && !hasSelfAccount()) {
-            throw new IllegalStateException("Node is not zero-stake, but has no known account");
+    public void validateSelfAccountIfNonZeroWeight() {
+        if (!isSelfZeroWeight() && !hasSelfAccount()) {
+            throw new IllegalStateException("Node is not zero-weight, but has no known account");
         }
     }
 
     /**
      * Returns true if the node in the address book at the given index (casting the argument as an
-     * {@code int}) has zero stake.
+     * {@code int}) has zero weight.
      *
      * @param nodeId the id of interest
-     * @return whether or not the node of interest has zero stake.
+     * @return whether or not the node of interest has zero weight.
      * @throws IllegalArgumentException if the {@code nodeId} cast to an {@code int} is not a usable
      *     index
      */
-    public boolean isZeroStake(long nodeId) {
+    public boolean isZeroWeight(long nodeId) {
```

Comment:
All the methods in this call should also specify `isZeroStake` even though we check `address.getWeight() <= 0;`

Reply:
Changed.